src/app/pages/send-crypto-asset/:fungible-token/stacks-fungible-token-send-form.tsx

```diff
@@ -0,0 +1,6 @@
+interface StacksFungibleTokenSendFormProps {
```

Comment:
if you haven't seen it yet, this file has `:` in its name

Reply:
This was intentional, indicating this is where the component goes when the dynamic route is hit (following the same naming convention as the router).

Not attached to this, just trying to make it clear that `stx` and `btc` are static, the other is dynamic.